movie.sh

```diff
@@ -30,38 +27,33 @@ function main(){
     # There should be exactly one argument i.e the address path
     if [ $NUM_ARGUMENTS != 1 ]; then
         add_style "This application takes one argument : Movie Directory" 1
-        rm -rf $TEMP_PATH
         exit 4
```

Comment:
I read through the exit codes 
1,2,125,126 etc were reserved
I did not find 4 as reserved so I chose it randomly